templates/solutions/automation/index.html

```diff
@@ -457,18 +454,17 @@ <h3 class="p-heading--5">Build a cloud with full transparency, through open sour
   {% endcall -%}
 
   <div class="p-section">
-    <div class="grid-row">
-      <div class="grid-col-medium-4 grid-col-start-large-5">
-        <div class="p-image-container">
-          {{ image(url="https://assets.ubuntu.com/v1/5c70b449-cloud%20native%20applications-mobile.png",
-                    alt="",
-                    width="864",
-                    height="576",
-                    hi_def=True,
-                    loading="lazy",
-                    attrs={"class": "p-image-container__image"}) | safe
-          }}
-        </div>
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-image-container">
+        {{ image(url="https://assets.ubuntu.com/v1/9280113e-cloud native applications.png",
```

Comment:
It might be safer to escape the spaces in the file name (same with the image below this).